lib/publiccloud/google.pm

```diff
@@ -79,4 +82,8 @@ sub upload_img {
     return $img_name;
 }
 
+sub cleanup {
+    record_info('cleanup');
```

Comment:
when there is nothing more maybe it little bit confusing : 
`record_info('cleanup','No cleanup needed');` or something similar

Reply:
I would not add it, if nothing to do.

Reply:
this is a placeholder, will add more content later on. If I don't put this here, upload_image.pm will fail as it calls $provider->cleanup();

Reply:
Really? It should not fail, cause the method is implemented in the parent provider.pm.